src/main/java/core/mate/academy/model/CustomException.java

```diff
@@ -0,0 +1,7 @@
+package core.mate.academy.model;
+
+public class CustomException extends RuntimeException {

```

Comment:
Give meaningful names for classes
```suggestion
public class CustomException extends RuntimeException {

```